_posts/dusan_andric/3001-01-01-intellij.md

```diff
@@ -0,0 +1,164 @@
+    image: dusan_andric/jetbrains/jetbrains.png
+header: 
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+## Introduction
+The fields of Cancer Research and Human Genomics, while presenting some very hard and interesting problems that touch the domains of Computer Science and Software Engineering, do not always have the resources to aquire or use the latest and greatest software tools. In particular it can be really difficult to justify such expenditures. 
+
+However, the kind folks at JetBrains were nice enough to support our development in these important fields, so as a thank you, we want to talk about what gets us so excited about using their tools. Choosing the right tools can significantly improve developer quality of life as well as productivity and we want to talk about the JetBrains tools that we love that do just that. 
+
+## IntelliJ IDEA
+
+IntelliJ is a very intuitive, robust, and dare I say it, enjoyable way of programming in Java. Both the community and Ultimate version include many features that accelerate the process of developing our software. While much of the appeal of IntelliJ is, as the cliche goes, that it just works, we highlight a few life-enhancing features that our team takes advantage of or is aided by. 
+
+### Duplicate Detection 
+
+IntelliJ can detect copy-and-paste code that someone lazily (cough) added into the code-base and allows you to quickly navigate between duplicates. 
+
+<figure>
+    <img src="{{site.urlimg}}dusan_andric/jetbrains/duplicates.png" />
+    <figcaption>duplicate code detection</figcaption>
+</figure>
+
+
+### Smart Quick Fixes
+
+IntelliJ can highlight areas that are potentially buggy or just non-optimal. A particularly nice touch is that pressing `Alt-Enter` will present a large number of context-sensitive options for fixing things. These options can range from suggesting a fix (that can be applied file or project wide), suggested improved code, annotations for hiding false positives, and configuring the check itself.
+
+<figure>
+    <img src="{{site.urlimg}}dusan_andric/jetbrains/quickfix.png" />
+    <figcaption>smart quick fixes</figcaption>
+</figure>
+
+### Context-sensitive Plugin Suggestion 
+
+So you're starting a new project or reinstalling your system. You open up a new file and IntelliJ suggests useful plugins which let IntelliJ understand types of code:
+
+<figure>
+    <img src="{{site.urlimg}}dusan_andric/jetbrains/detection.png" />
+    <figcaption>context-sensitive plugin suggestion</figcaption>
+</figure> 
+
+It's a small touch, but appreciated. 
+
+### Code-style Importing 
+
+So you like IntelliJ but your team likes Eclipse. Or vice versa. Keep things in sync and reduce un-necessary or distracting formatting challenges by taking advantage of IntelliJ's ability to import code formatting files from Eclipse, from CheckStyle, or from other IntelliJ installs. 
```

Comment:
> from Eclipse, from CheckStyle, or from other IntelliJ installs. 
 
recommended to change to : `from Eclipse, CheckStyle, or other IntelliJ installations.`

DRY :D